docs/UserGuide.md

```diff
@@ -921,32 +1170,52 @@ If you wish to remove a scheduled meetup from your [Scheduled Meet Up Panel](#sc
 
 </div>
 
-<div markdown="block" class="alert alert-success">
 
-:bulb: **Example : Removing a meet up from the scheduled meet up list**<br>
-* `unorganise 1`: Removes the scheduled meet up with index 1 on the [Scheduled Meet Up Panel](#scheduled-meet-up-panel).
+##### Scenario:
 
-</div>
+Suppose Tae Seong has got Covid-19 and cannot make it to the meet up. Unfortunately, you have to cancel the meet up.
+
+The scheduled meet up has the index 1 on the [Scheduled Meet Up Panel](#scheduled-meet-up-panel).
```

Comment:
Too many occurrences of "meet up" in quick succession. Perhaps spacing them out a bit.

"Suppose Tae Seong cannot attend the meet up due to contracting Covid-19, and you have to cancel the scheduled meet up."

This way, "meet up" are further apart.